must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -60,6 +62,8 @@ commands_desc+=("storagecluster")
 commands_desc+=("volumesnapshot -A")
 commands_desc+=("volumesnapshotclass")
 commands_desc+=("volumesnapshotcontent")
+commands_desc+=("storageconsumer")
+commands_desc+=("cephfilesystemsubvolumegroups.ceph.rook.io")
```

Comment:
Need to remove from here and line 55 as well? i don't see we are doing the same for cephblockpool. can you please confrm

Reply:
yeah we aren't, but as I mentioned that @nehaberry wants to collect output for oc get and desc commands which is not done by ceph_resources. Neha has mentioned the details in bug description.
Do you think that is not required and we can remove that as well?

Reply:
@nehaberry do we need to treat this one more special that other ceph resources?